README.md

```diff
@@ -26,7 +26,7 @@ Documentation for Stan math is available at [mc-stan.org/math](https://mc-stan.o
 
 Installation
 ------------
-The Stan Math Library is a C++ library which depends on the Intel TBB library and requires for some functionality (ordinary differential equations and root solving) on the Sundials library.
+The Stan Math Library is a C++ library which depends on the Intel TBB library and requires for some functionality (ordinary differential equations and root solving) on the Sundials library. As build system the make facility is used to manage all dependencies.
```

Comment:
I would say: "As build system, we use the make facility to manage all dependencies."

Reply:
I think we never use "we" in this file, so I would not start to use it, but have a look at the rewording.

Reply:
I hadn't noticed that, but I agree with you.